client/src/router.tsx

```diff
@@ -124,8 +131,18 @@ const Router = ({ appConfig }: Props) => {
   );
 
   useEffect(() => {
-    const adminUser = AdminTokenStore.get();
-    setAdmin(adminUser);
+    const adminToken = AdminTokenStore.get();
+    setAdmin(adminToken);
+
+    // TODO: read also beacon token from local store
+
+    const fetchPublicBeacons = async () => {
+      const pubBeacons = await ShareLocationApi.fetchPublicBeacons();
```

Comment:
Currently the tests throw an UnhandledPromiseRejectionWarning with the message "connect ECONNREFUSED 127.0.0.1:3001".

This problem disappears if I wrap the contents of fetchPublicBeacons inside a try...catch. I wonder, though, if there's some underlying cause which merits attention?

Reply:
It's probably related to the test runs having different environment, maybe API urls are wrong or something. You can use `it.skip` function to skip tests. I'd avoid commenting out code or adding try/catch blocks for just test runs.

Reply:
It's probably related to differences in test environment, maybe API urls are wrong or something like that. You can skip tests with `it.skip`, i'd avoid commenting out code or adding try/catch blocks for just test runs.